bedrock/newsletter/templates/newsletter/knowledge-is-power.html

```diff
@@ -0,0 +1,54 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends 'base-protocol.html' %}
+
+{% block page_title %}{{ ftl('newsletters-subscribe-to-the-newsletter') }}{% endblock page_title %}
+
+{% block page_desc %}{{ ftl('newsletters-get-the-news-and-tips')}}{% endblock %}
+
+{% block body_class %}{{ super() }} newsletter-knowledge-is-power{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('protocol-newsletter') }}
+  {{ css_bundle('newsletter-knowledge-is-power') }}
+{% endblock %}
+
+{% block js %}
+  {{ js_bundle('newsletter') }}
+{% endblock %}
+
+{% block content %}
+<main role="main mzp-t-content-sm">
```

Comment:
**issue:** invalid role, and unnecessary since you've got the native `main` element in place

```suggestion
<main class="mzp-t-content-sm">
```

Reply:
🦅 👁️ !